### PR TITLE
Improve dropdown menus, remove inline styles

### PR DESCRIPTION
**Pull request by silverwind**
Before:

Screenshot 2023-05-27 at 10 46 43

After:

Screenshot 2023-05-28 at 19 29 03

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Comment by wxiaoguang** [comment]
How about just removing these "headers" in the dropdown menu?

I don't find these headers provide useful information. Without them, the UI looks better to me:


image


image

**Comment by silverwind** [comment]
Yeah I had the same feeling, they are pretty pointless in general, also in the reaction dropdown.

**Comment by silverwind** [comment]
Ok I have removed all titles from issue sidebar dropdowns now. There are a few remaining cases where title is still used for "No Items", I guess I'll leave those.

**Review by delvh — APPROVED** [review]

File: templates/repo/issue/labels/labels_selector_field.tmpl [hunk 1]
```diff
@@ -37,7 +36,7 @@
 				{{if .Description}}<br><small class="desc">{{.Description | RenderEmoji $.Context}}</small>{{end}}</a>
 			{{end}}
 		{{else}}
-			<div class="header" style="text-transform: none;font-size:14px;">{{.locale.Tr "repo.issues.new.no_items"}}</div>
+			<div class="header header-no-uppercase">{{.locale.Tr "repo.issues.new.no_items"}}</div>
```
Comment (wxiaoguang) [inline templates/repo/issue/labels/labels_selector_field.tmpl]: According to official document, it should be a "disabled item" https://fomantic-ui.com/modules/dropdown.html#disabled

image

Comment (silverwind) [reply templates/repo/issue/labels/labels_selector_field.tmpl]: Ah yes, much better than abusing a header.

**Comment by silverwind** [comment]
Now using `disabled item`:

Screenshot 2023-05-28 at 19 29 03

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**